src/components/filter/InputFilter.js

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import classes from './InputFilter.module.css';
+
+const InputFilter = ({ setSearched }) => {
```

Comment:
Good job for implementing the search filter, however, it needs further improvement.
- [ ] When part of a search term is keyed in the search field, the filter result does not display all possible matches. For instance, when I enter `eth` in the search field, the only displayed result is `Tether` whereas other coins such as `Ethereum` and `Lido Staked Ether` also have the term `eth` in their names. Kindly ensure to improve upon this aspect.

**Searching with the term `eth`:**
![image](https://user-images.githubusercontent.com/12745474/142268552-a2eeaf9c-fdd0-4ca1-8055-8dcb98aaa16e.png)